forem/forem
# /new form title fix font font-size

## What type of PR is this? (check all applicable)


## Description

This will fix two bugs:

1. Wrong font size in title field on /new
2. Jump while loading the page because of .html.erb and preact differences.

## Related Tickets & Documents

#12572

## QA Instructions, Screenshots, Recordings

Go to `/new` on mobile and notice NO jump of title field font-size. It should also be bigger than body textarea font-size.

**Before:**
![image](https://user-images.githubusercontent.com/108287/106926677-a11bf100-6711-11eb-9c99-35df290a6e6c.png)

**After:**
![image](https://user-images.githubusercontent.com/108287/106926744-b4c75780-6711-11eb-96de-4d3a7270de20.png)

### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested. For more info, check out the
[Forem Accessibility Docs](https://docs.forem.com/frontend/accessibility)._

## Added tests?


## Added to documentation?

      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/)
